Sami/Parser/ClassVisitor/PropertyClassVisitor.php

```diff
@@ -50,58 +49,26 @@ public function visit(ClassReflection $class)
      * Adds a new property to the class using an array of tokens.
      *
      * @param ClassReflection $class       Class reflection
-     * @param string          $propertyTag Property tag contents
+     * @param string|array    $propertyTag Property tag contents
      *
      * @return bool
      */
     protected function injectProperty(ClassReflection $class, $propertyTag)
```

Comment:
If you're only injecting array from all places, then you can:

* add `array` type hint 
* update type in DocBlock from `string|array` into `array`